File: Java/String/program2.java
//Accept the string and display each character

import java.util.*;

class program2
{
    public static void main(String arg[])
    {
        String str = "Hello";
        
        char Arr[] = str.toCharArray();

        for(int i = 0; i < Arr.length; i++)
        {
            System.out.println(Arr[i]);
        }
    }
}

File: C/String/program2.c
//Display the string

#include<stdio.h>

int main()
{
    char Arr[] = "Hello";
    int i = 0;

    printf("The string is: \n");

    while(Arr[i] != '\0')
    {
        printf("%c",Arr[i]);
        i++;
    }

    return 0;
}
